notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -45,8 +45,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For example, usually, the data that appears in recommendation problems can be encoded in the following way - the user, item, and feature vectors are transformed into one-hot representation for building model. For this arrangement, using the classic algorithms like linear regression, SVM, for predicting the target, are that\n",
-    "1. The feature vectors are highly sparse, and thus it makes it hard to converge the parameters to fit the model efficienly\n",
+    "In certain occasions, the data that appears in recommendation problems, such as user, item, and feature vectors, can be encoded into a one-hot representation. Under this arrangement, the use the classical algorithms like linear regression, SVM, have from the following problems:\n",
```

Comment:
Thanks for addressing this! I really don't want to be a picky person, but
> the use the classical algorithms like linear regression, SVM, have from the following problems:

"classical algorithms like linear regression and SVM may suffer from the following problems:"

Reply:
you are not picky, you do care